README.md

```diff
@@ -31,7 +31,7 @@ It allows you to easily create, maintain and execute automated web tests across
 
 Tests             | Status
 ----------------- | ----------
-All               | [![Build Status](https://travis-ci.org/DevExpress/testcafe.svg)](https://travis-ci.org/DevExpress/testcafe)
+All               | [![Travis CI Status](https://travis-ci.org/DevExpress/testcafe.svg)](https://travis-ci.org/DevExpress/testcafe) [![AppVeyor status](https://ci.appveyor.com/api/projects/status/ftelkyuiji8lyadf?svg=true)](https://ci.appveyor.com/project/DevExpress/testcafe)
```

Comment:
Just put them in separate rows:
- For Travis: `All Travis tasks (server, client, functional mobile, etc.)`
- For AppVeyor `Functional desktop`